limitador-server/sandbox/README.md

```diff
@@ -17,14 +17,39 @@ Check out `make help` for all the targets.
 
 ### Deployment options
 
-| Limitador's configuration | Command | Info |
-| ------------- | ----- | ----- |
-| In-memory configuration | `make deploy-in-memory` | Counters are held in Limitador (ephemeral) |
-| Redis | `make deploy-redis` | Uses Redis to store counters |
-| Redis Secured | `make deploy-redis-tls` | Uses Redis with TLS and password protected to store counters |
-| Redis Cached | `make deploy-redis-cached` | Uses Redis to store counters, with an in-memory cache |
-| Redis Otel Instrumented | `make deploy-redis-otel` | Uses redis to store counters, [instrumented with opentelemetry](redis-otel/README.md) |
-| Disk | `make deploy-disk` | Uses disk to store counters |
+| Limitador's configuration | Command | Info                                                                                                           |
+|--------------------------| ----- |----------------------------------------------------------------------------------------------------------------|
+| In-memory configuration  | `make deploy-in-memory` | Counters are held in Limitador (ephemeral)                                                                     |
+| Redis                    | `make deploy-redis` | Uses Redis to store counters                                                                                   |
+| Redis Secured            | `make deploy-redis-tls` | Uses Redis with TLS and password protected to store counters                                                   |
+| Redis Cached             | `make deploy-redis-cached` | Uses Redis to store counters, with an in-memory cache                                                          |
+| Redis Otel Instrumented  | `make deploy-redis-otel` | Uses redis to store counters, [instrumented with opentelemetry](redis-otel/README.md)                          |
+| Disk                     | `make deploy-disk` | Uses disk to store counters                                                                                    |
+| Distributed | `make deploy-distributed` | Counters are held in Limitador (ephemeral) but replicated to other Limitador servers. |
+
+
+### Running Multi Node Distributed Deployments
+
+The `make deploy-distributed` target can be connected to other Limitador servers but requires you to set the `PEER_ID` and `PEER_URLS` environment variables when you run the target.
```

Comment:
have you considered running 3 instances (or N) of limitador on the same docker compose instance?

Reply:
I guess I could add more variables for the ports.. might get challenging exposing all the ports on the host right?

Reply:
you do not need to worry about port conflict. Each limitador instance will be a unique container with dedicated assigned IP in the assigned docker network. Different IPs.. no port conflict. Docker magic.